crates/driver/src/infra/config/file/mod.rs

```diff
@@ -431,6 +433,8 @@ enum BalancerV2Config {
         /// Deny listed Balancer V2 pools.
         #[serde(default)]
         pool_deny_list: Vec<eth::H256>,
+
+        graph_api_base_url: Url,
```

Comment:
Preset pools (which are used in prod) should also be able to configure this value

Reply:
I have actually moved the config to the LiquidityConfig so I think that this comment is outdated now.